model_compression_toolkit/gptq/keras/quantizer/ste_rounding/symmetric_ste.py

```diff
@@ -27,7 +27,34 @@
 from model_compression_toolkit.gptq.common import gptq_constants
 
 
-def symmetric_constrained_quantizer(input_tensor: tf.Tensor,
+def symmetric_quantizer(input_tensor: tf.Tensor,
```

Comment:
Lots of code duplication between this and pertubation_symmetric_quantizer.
Consider extracting the shared code to a helper function (it'll help to highlight the differences between the quantizers)